launch/app/pom.xml

```
@@ -55,7 +55,14 @@
       <version>${project.version}</version>
       <scope>runtime</scope>
     </dependency>
-    
+
+    <dependency>
+      <groupId>org.openhab.addons.bundles</groupId>
```

We should sync the GAV naming scheme with the other boms - the groupID ends with "bom" there.
```suggestion
      <groupId>org.openhab.addons.bom</groupId>
```